pandas/core/ops.py

```diff
@@ -594,20 +594,26 @@ def wrapper(self, other):
 
         mask = isnull(self)
 
-        values = self.get_values()
-        other = _index.convert_scalar(values,_values_from_object(other))
+        if com.is_categorical_dtype(self):
+            # cats are a special case as get_values() would return an ndarray, which would then
```

Comment:
in reaility some of the ``core/ops.py`` *should* be in internals. @shoyer let's make an issue about this, but @JanSchulz change is ok for now.

@JanSchulz can you add a release note. I'll merge this before ``CategoricalIndex``